### ISSUE TITLE
Live Preview reloads too often and not fully

### ISSUE THREAD

**Issue by dmetzler1988** [sulu/sulu#3540]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.6.3
| Browser Version | Google Chrome 61.0.3163.100

#### Actual Behavior

At first:
If you open a page on backend (or reload it) the website will be shown on the live preview window on right side. After all elements on left side (contents) are loaded, the live preview window will make a reload again.

If you take a look to the XHR requests, the second `admin` request will shown this message:
```
{"handler":"sulu_preview.preview","message":{"command":"update-context","webspaceKey": ...
```

At second:
If you reload the preview window, the page will be refreshed. But the JS will be initialized again - without removing the previous instance.
If you are defining global JS elements, these are duplicated.

#### Expected Behavior

The live preview should be loaded at once - not twice. It is a performance issue.
Also the whole page should be reloaded - to solve the issue with duplicated JS elements.

#### Steps to Reproduce

1. Open a page on backend. 
2. Set a breakpoint to the own JS file.
3. Reload the page
4. Now the breakpoint should be triggered twice.
5. Check the XHR request which was triggered and take a look to the message of the second one.

**Comment by wachterjohannes** [comment]
@dmetzler1988 will be fixed with the next HOTFIX release.

**Comment by danrot** [comment]
fixed in #3543
